git_creation_and_tracking/git_creation_and_tracking.md

```diff
@@ -0,0 +1,810 @@
+```
+
+Dracula has created two tracking files, `planets/.git` which he should keep, and `planets/moons/.git` which should be removed.
+Dracula can do this by running `pwd` which will show the **present working directory** and getting the output:
+
+```console
+/Users/vlad/Desktop/planets
+```
+before running:
+
+```console
+$ rm -rf moons/.git
+```
+<div class = "warning">
+The `rm` command **removes** files, and can be very useful, but the `-rf` flag is very powerful and should used with great caution. The `r` stands for "recursive" and instructs the computer to remove all contents of the file and then remove the file itself.  The `f` is for "force" and means you will not be asked if you are sure you want to delete the files.
+
+Dracula should be absolutely certain he wants to delete `moons/.git` before running this command.
+</div>
+
+## Tracking changes
+
+ It might seem like **making** changes and **documenting** them are two different things, but the entire point of version control is to document every change you make.
+
+ Just like any system that can help back-up or save your work, Git will only be helpful if you use it. In this section we will first teach you
+
+ - **how** to make a record of your changes,
+
+ - **where** those records are, and
+
+ - **when** you should be recording that you changed things.
+
+### Tell Git to track a file
+Let's follow along with Dracula to add a file to the planets directory with notes about Mars.
+
+First let's make sure we're still in the right directory.
+You should be in the `planets` directory.
+
+```console
+$ cd ~/Desktop/planets
+```
+
+Let's create a file called `mars.txt` that contains some notes
+about the Red Planet's suitability as a base.
+We'll use `nano` to edit the file;
+you can use whatever editor you like.
+In particular, this does not have to be the `core.editor` you set globally earlier. But remember, the bash command to create or edit a new file will depend on the editor you choose (it might not be `nano`). For a refresher on text editors, check out ["Which Editor?"](https://swcarpentry.github.io/shell-novice/03-create/) in [The Unix Shell lesson](https://swcarpentry.github.io/shell-novice/) by Software Carpentry.
+
+```console
+$ nano mars.txt
+```
+
+Type the text below into the `mars.txt` file:
+
+
+```console
+Cold and dry, but everything is my favorite color
+```
+
+
+Let's first verify that the file was properly created by running the list command (`ls`):
+
+
+```console
+$ ls
+
+mars.txt
+```
+
+
+
+`mars.txt` contains a single line, which we can see by running `cat`:
+
+
+```console
+$ cat mars.txt
+
+Cold and dry, but everything is my favorite color
+```
+
+
+If we check the status of our project again,
+Git tells us that it's noticed the new file:
+
+```console
+$ git status
+
+On branch main
+
+No commits yet
+
+Untracked files:
+   (use "git add <file>..." to include in what will be committed)
+
+	mars.txt
```

Comment:
It looks like you can't style things inside a code block because it reads everything verbatim. Which makes sense because that is what we mean by making it a code block. It seems like a significant amount of work to make a new code block environment in css to do this, so I think the answer is no.